resources/js/features/achievements/components/AchievementGroup/AchievementGroup.tsx

```diff
@@ -0,0 +1,23 @@
+import type { FC } from 'react';
+
+import { UnlockableAchievementAvatar } from '@/features/achievements/components/UnlockableAchievementAvatar/UnlockableAchievementAvatar';
```

Comment:
nit: You should be able to import without the double-path:
```ts
import { UnlockableAchievementAvatar } from '@/features/achievements/components/UnlockableAchievementAvatar';
```
This can be done by creating an _index.ts_ file in the folder and exporting the component:
```
export * from './UnlockableAchievementAvatar';
```
This can seem trivial, but it's good for enforcing module boundaries of a component to prevent anything else in the _UnlockableAchievementAvatar_ folder from accidentally "leaking" during `pnpm build`'s compile-time (ie: test code, utils, etc). Think of the _UnlockableAchievementAvatar_ folder as a module, and _index.ts_ declares its public API (each thing exported is publicly-available).

Reply:
Done.

Pretty sure VSCode auto-added that for me.

The `index.ts` file is present - I started by copy/pasting all the files from another widget

Also, VSCode insists on reformatting all my indents to four spaces, so I have to `eslint -- fix` them back to two before committing.

Reply:
Highly recommend installing the VSCode ESLint extension [here](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and then enabling VSCode's format on save.

Open the command palette with Ctrl+Shift+P, then open "Preferences: Open User Settings (JSON)" and add these lines:
```json
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "editor.formatOnSave": true,
  "editor.codeActionsOnSave": {
    "source.fixAll.eslint": "explicit"
  },
```

Now in .ts and .tsx files, when you save, ESLint will just run and fix everything that has an auto-fixer. For me, this is a significant time-saver.